Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:alpine
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN adduser -S sopel
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
+
+USER sopel
+
+# OS X users will crash with permission errors if they use a
```

Comment:
Is this still valid since the transition from boot2docker to Docker Machine for Mac?